repos/fdbt-site/src/pages/products/pointToPointProducts.tsx

```diff
@@ -110,9 +110,12 @@ export const getServerSideProps = async (
     const formattedProducts = await Promise.all(
         productsToDisplay.map(async (product) => {
             const matchingJson = await getMatchingJson(product.matchingJsonLink);
-            const productDescription = `${upperFirst(matchingJson.passengerType)} - ${matchingJson.type} ${
+            let productDescription = `${upperFirst(matchingJson.passengerType)} - ${matchingJson.type} ${
                 matchingJson.carnet ? '(carnet)' : ''
             }`;
+            if (matchingJson.type === 'period' && 'products' in matchingJson) {
+                productDescription = matchingJson.products[0].productName;
+            }
```

Comment:
Can we ternary this to avoid the let and redundant assignment?

Reply:
Yep, good call